src/App.js

```diff
@@ -1,12 +1,85 @@
 import React from 'react';
 import './App.css';
+import bag_1 from './assets/bag_1.png';
+import bag_2 from './assets/bag_2.png';
+import bag_3 from './assets/bag_3.png';
+import bag_4 from './assets/bag_4.png';
+import brand from './assets/brand.png';
+import our_story from './assets/our_story.png';
+import Button from "./Components/Button";
+import Product from "./Components/Product";
+import Tile from "./Components/Tile";
 
 function App() {
   return (
-      <h1>Handbags & Purses</h1>
+<>
+        <h1>Handbags & Purses</h1>
+      <nav>
+          <Button
+              typeOfButton="button"
+              clickHandler={() => console.log("to the collection")}
+              toggleDisabled={false}
+              buttonName="to the collection"
+          />
+          <Button
+              typeOfButton="button"
+              clickHandler={() => console.log("shop all bags")}
+              toggleDisabled={false}
```

Comment:
Volgens mij is een "disabled"-waarde van een `<button>` by default `false` en hoef je deze dus niet als prop mee te geven aan je `<Button>` tenzij je hem de waarde `true` wilt geven.